iosxr_iso2vbox.py

```diff
@@ -326,12 +362,13 @@ def xr_cli_wait_for_output(command, pattern):
             child.sendline("ssh server vrf default")
             child.expect("config")
 
-        # Configure gRPC protocol if MGBL package is available
-        if mgbl:
-            child.sendline("grpc")
-            child.expect("config-grpc")
-            child.sendline(" port 57777")
-            child.expect("config-grpc")
+        # We no longer enable gRPC by default as it consumes a large
```

Comment:
You think this is a transitional issue or it's going to stay this way? Just curious.